os/board/imxrt1050-evk/src/Makefile

```diff
@@ -88,7 +88,7 @@ CSRCS += partitions.c
 endif
 
 ifeq ($(CONFIG_BOARD_CRASHDUMP),y)
-CSRCS += crashdump.c
+CSRCS += imxrt_crashdump.c
```

Comment:
why have to move from common crash dump to board specific?

Reply:
All boards might not support an external ram which needs to be dumped. Also we have used board specific configs for external ram

Reply:
It was moved to common for making generic implementation.
(CONFIG_IMXRT_SEMC_SDRAM), CONFIG_IMXRT_SDRAM_START, CONFIG_IMXRT_SDRAM_SIZE
I see these configs are imxrt specific, can we rename or find method to make it common name and use by all boards?

Reply:
As @sangam-swami said, please make it as common.

Reply:
Updated in common crashdump file itself using CONFIG_IMXRT_SEMC_SDRAM